<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailInteraction</class>
 <widget class="QDialog" name="DetailInteraction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>426</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>764</width>
    <height>426</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>764</width>
    <height>426</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>OGC (Outil de Gestion de Contacts)</string>
  </property>
  <widget class="QLabel" name="titre">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>761</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Détails de l'interaction</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="champTitreInte">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>543</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitreInteraction">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>80</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre de l'interaction :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDerniereModif">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dernière modification :</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateDerniereModif">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="validerModifs">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>370</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider les modifications </string>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonRetour">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="champContenuInte">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>713</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
